services/hackernews/hackernews-user-karma.service.js

```diff
@@ -0,0 +1,71 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { BaseJsonService, NotFound } from '../index.js'
+import { anyInteger } from '../validators.js'
+
+const schema = Joi.object({
+  id: Joi.string(),
+  karma: anyInteger,
+})
+  .allow(null)
+  .required()
+
+export default class HackerNewsUserKarma extends BaseJsonService {
+  static category = 'social'
+
+  static route = {
+    base: 'hackernews/user-karma',
+    pattern: ':userid',
+  }
+
+  static examples = [
+    {
+      title: 'HackerNews User Karma',
+      namedParams: { userid: 'drumstick' },
+      staticPreview: this.render({ userid: 'drumstick', karma: 15536 }),
+    },
+  ]
+
+  static defaultBadgeData = {
+    label: 'HackerNews User Karma',
```

Comment:
This is a bit of a mouthful for a label, and I'd argue, going against the grain of the [branding/promotional clause in our badge specification](https://github.com/badges/shields/blob/master/spec/SPECIFICATION.md#example). There are a few cases where our default badge label has the service name included because we felt it was necessary to qualify/describe the value in the message.

However, I'm not sure that's the case here. Consider in reference our other social-style badges, including the Reddit Karma badge which is `combined karma` (I'm not sure what "combined" means in that context, but the main point is that the label doesn't include "Reddit").

https://shields.io/category/social

I think we should change this to:

```suggestion
    label: 'User Karma',
```

or

```suggestion
    label: 'Karma',
```